src/@next/Menu/Menu.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useCallback } from 'react';
+import { Typography } from '../Typography';
+import { Neutral } from '../utilities/colors';
+import { ListContainer, StyledMenu, TitleContainer } from './MenuStyle';
+
+interface Option {
+  active?: boolean;
```

Comment:
Hmm wouldn't this redundant with selected props? In what case user need to supply active?

Reply:
you're right! let me remove this too